pkgs/applications/emulators/duckstation/default.nix

```diff
@@ -44,55 +41,55 @@ stdenv.mkDerivation rec {
   buildInputs = [
     SDL2
     curl
+    gtk3
+    libevdev
     libpulseaudio
-    libXrandr
     mesa
     qtbase
-    qtsvg
+    sndio
     vulkan-loader
-    wayland
   ];
 
   cmakeFlags = [
     "-DUSE_DRMKMS=ON"
-    "-DUSE_WAYLAND=ON"
   ];
 
-  desktopItems = [
-    (makeDesktopItem {
-      name = "DuckStation";
-      desktopName = "JamesDSP";
-      genericName = "PlayStation 1 Emulator";
-      icon = "duckstation";
-      tryExec = "duckstation-qt";
-      exec = "duckstation-qt %f";
-      comment = "Fast PlayStation 1 emulator";
-      categories = [ "Game" "Emulator" "Qt" ];
-    })
-  ];
+  postPatch = ''
+    substituteInPlace extras/linux-desktop-files/duckstation-qt.desktop \
+      --replace "duckstation-qt" "duckstation" \
+      --replace "TryExec=duckstation" "tryExec=duckstation-qt" \
+      --replace "Exec=duckstation" "Exec=duckstation-qt"
+    substituteInPlace extras/linux-desktop-files/duckstation-nogui.desktop \
+      --replace "duckstation-nogui" "duckstation" \
+      --replace "TryExec=duckstation" "tryExec=duckstation-nogui" \
+      --replace "Exec=duckstation" "Exec=duckstation-nogui"
```

Comment:
Why do we need to fix those?

Reply:
It'll get removed next version (which removes the desktop files completely from the repo, meaning they'll be created with `makeDesktopItems`).